Nodejs/Tests/Core.UI/BasicProjectTests.cs

```diff
@@ -993,6 +993,20 @@ public class BasicProjectTests : SharedProjectTest {
                 actual = 0;
             }
             Assert.AreEqual(expected, actual, "count[" + key + "]");
+        }
+
+        /// <summary>
+        /// Remove the label denoting browser-side code, node, or both depending on the content type.
+        /// </summary>
+        private static string removeLabel(string itemName) {
+            string[] labels = new string[3] { " (browser)", " (node)", " (node, browser)" };
+            foreach (var label in labels) {
+                if (itemName.EndsWith(label)) {
+                    itemName = itemName.Substring(0, itemName.Length - label.Length);
+                    break; // safe to break as there is at most one label per name
+                }
+            }
+            return itemName;
```

Comment:
After a second look, I will remove this change as it is not related to the three test cases I am fixing in this pull request (the only test case that is affected by this change is BasicProjectTests\LoadProjectWithDuplicateItems and it still fails after this change).